code/src/components/DescriptionContainer.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const DescriptionContainer = props => {
+  return (
+    <article className="album-text-container">
+      <h2 className="album-title">
+        <a target="_blank" rel="noopener noreferrer" href={props.hrefAlbum}>
+          {props.title}
+        </a>
+      </h2>
+      <p className="artists">
+        {props.singer.map(artists => (
+          <a
+            key={artists.id}
+            target="_blank"
+            rel="noopener noreferrer"
+            href={artists.external_urls.spotify}
+          >
+            <span>{artists.name}</span>
```

Comment:
Love to know how you managed to get the artist seperated, we managed to do that, but it looks like only one link. GREAT JOB! 💫